src/global.zig

```diff
@@ -170,9 +176,18 @@ pub const GlobalState = struct {
 
         // Find our resources directory once for the app so every launch
         // hereafter can use this cached value.
-        self.resources_dir = try internal_os.resourcesDir(self.alloc);
+        self.resources_dir = rd: {
+            if (@hasDecl(apprt.runtime, "resourcesDir")) break :rd try apprt.runtime.resourcesDir(self.alloc, false);
```

Comment:
We try to avoid `@hasDecl` in new apprt-related code, although I'm not exactly sure what's a better alternative here